File: Clock/Counter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClockTask
{
    public class Counter
    {
        private long _count;
        private string _name;

        public Counter(string name) 
        {
            _name = name;
            _count = 0;
        }

        public void Increment() 
        {
            _count += 1;
        }

        public void Reset()
        {
            _count = 0;
        }

        public string Name
        {
            get 
            { 
                return _name;
            }
            set 
            { 
                _name = value;
            }
        }

        public long Ticks 
        {
            get
            {
                return _count;
            }
        }

        public long ResetByDefault
        {
            set
            {
                _count = 2147483647358;
            }
        }


    }
}

File: Clock_python/counter.py
class Counter:
    def __init__(self, name):
        self._count = 0
        self._name = name

    def increment(self):
        self._count += 1

    def reset(self):
        self._count = 0

    @property
    def name(self):
        return self._name

    @name.setter
    def name(self, value):
        self._name = value

    @property
    def ticks(self):
        return self._count

    @property
    def reset_by_default(self):
        return 2147483647358  # just for compatibility, not actually used
